packages/babel-helper-compilation-targets/src/options.js

```diff
@@ -14,7 +15,4 @@ export const TargetNames = {
   android: "android",
   electron: "electron",
   samsung: "samsung",
-
-  // deprecated
-  uglify: "uglify",
```

Comment:
In practice, we handle this by deleting `targets.uglify` before using this object.